base.css

```diff
@@ -0,0 +1,66 @@
+
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, h1, h2, h3, h5, h6, p,
+a,img,th, td,article, aside,
+footer, header, hgroup,nav,section{
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol, ul {
+  list-style: none;
+}
+blockquote, q {
+  quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/*personal styles*/
+
+h3 {
+  font-family: "Oswald",bold;
+  font-size: 42px;
+  margin: 0px;
+}
+h5 {
+  font-family: 'Abel';
+  margin-top: 0px;
+  color: grey;
+}
+ul{
+  list-style-type: none;
+}
+article p {
+  text-indent: 30px;
+}
+article {
+  font-family: 'Droid serif';
+  font-size: 20px;
+}
+.chocolateCake img{
```

Comment:
move this into your modules.css. base should be only for general rules that apply universally to your site.